### DISCUSSION TITLE
How to add a smaller date to a larger grid, or how to interpolate it onto it

### DISCUSSION THREAD

**Discussion by Gleb-GG** [pyvista/pyvista#1636] [Q&A] [answered]
Hi, pyvista team,

I have a 3d model of the surface relief as I did in the example https://docs.pyvista.org/examples/00-load/terrain-mesh.html#sphx-glr-examples-00-load-terrain-mesh-py
![image](https://user-images.githubusercontent.com/79473992/132098831-87f6f019-6a54-4cb6-9bfc-c9cf6ab61475.png)

Data of model 
![image](https://user-images.githubusercontent.com/79473992/132099063-9ddd79be-0b91-4903-8087-077ea185c391.png)

I want to add an array with seismic tomography there, but its size is much smaller than this model

How do I interpolate the values ​​of a smaller array to points in the model with a larger date? I add this array with the command

mesh.point_arrays["R"] = data.flatten(order="F")

**Comment by Gleb-GG** [comment, marked as answer]
I found a solution, if you have a date with a smaller size, increase it with

```py
from scipy.ndimage import zoom
new_array = zoom(P, (7.14, 1.92, 1))
```

**Reply by akaszynski** [reply]
Thanks for posting the solution (been busy with #1571 and #1557 over the weekend).
